### PR TITLE
Add no data message for cell type wheel

### PR DESCRIPTION
**Pull request by lingyun1010**

### CODE REVIEW

File: packages/scxa-cell-type-wheel/src/CellTypeWheel.js [hunk 1]
```diff
@@ -4,10 +4,16 @@ import PropTypes from 'prop-types'
...
 
 import _ from 'lodash'
 
-highchartsSunburst(Highcharts)
+async function addModules () {
+  HighchartsNoData(Highcharts)
+  highchartsSunburst(Highcharts)
+}
+
+addModules()
```
Comment (upendrakumbham) [inline packages/scxa-cell-type-wheel/src/CellTypeWheel.js:16]: Do you need this `addModules()` function call here?

File: packages/scxa-cell-type-wheel/src/CellTypeWheel.js [hunk 2]
```diff
@@ -46,6 +52,17 @@ class CellTypeWheel extends React.Component {
           }
         },
 
+        lang: {
+          noData: `${props.species} doesn't have wheel data for the organism part: ${props.searchTerm}.`
```
Comment (ke4) [inline packages/scxa-cell-type-wheel/src/CellTypeWheel.js:56]:
```suggestion
          noData: `No data exists in our database for this search term: ${props.searchTerm} for ${props.species}.`
```

**Review by ke4 — APPROVED** [review]
I just suggested to change the `noData` message, but otherwise looks fine by me

**Review by upendrakumbham — COMMENTED** [review]
only tiny clarification on this `addModule()`, otherwise all looks good to me to approve.

File: packages/scxa-cell-type-wheel/src/CellTypeWheel.js (hunk 2 above)
Comment (lingyun1010) [reply packages/scxa-cell-type-wheel/src/CellTypeWheel.js:56]: I don't know, I think that was agreed in the team meeting, so @upendrakumbham wrote it down in the issue. https://github.com/ebi-gene-expression-group/atlas-web-single-cell/issues/455#issue-2478240961

**Pull request merged**